### PR TITLE
Feat/friend page

### LINKED ISSUE #9: Design Central Framework - Friend Page
# Value Statement
As a user of the App, I want to create a responsive flowing design so that users can easily navigate the friends page in a natural way.

# Description

Screenshot 2021-04-27 at 15 47 24

# Acceptance Criteria

- The friend page will have a search icon, the friends in a cloud formation and a background image of some kind.
- Clicking on a friend takes you to their respective pages.

# Tasks
- [x] Setup stateless elements
- [x] Setup multiple Component files in a central `components` folder.
- [x] Set the text styling.
- [x] Set a background image.
- [x] Use the SVGs from Adobe XD for the clouds.
- [x] Clouds can repeat after 5 shapes.
- [x] use random images of dogs for the friend icons.

# Complexity

Medium

### PR DESCRIPTION
**Pull request by hugh-burgess**
resolves #9

### CODE REVIEW

File: client/src/components/friends-page/FriendsContent.css [hunk 1]
```diff
@@ -16,7 +15,7 @@
 }
 
 .clouds-wrapper {
-  height: 100%;
+  height: 90vh;
   display: flex;
   flex-direction: column;
   width: 100vw;
```
Comment (soerendolling) [inline client/src/components/friends-page/FriendsContent.css]: Just the Monk inside me. You might wanna stick to a order within your css rulsesets.
like
    From outside to insde
    Layout Properties (position, float, clear, display)
    Box Model Properties (width, height, margin, padding)
    Visual Properties (color, background, border, box-shadow)
    Typography Properties (font-size, font-family, text-align, text-transform)
    Misc Properties (cursor, overflow, z-index)

**Review by soerendolling — APPROVED** [review]
lgtm

File: client/src/components/friends-page/FriendsContent.css [hunk 2]
```diff
@@ -56,3 +61,29 @@
...
   z-index: 100;
 }
+
+.names-left {
+  font-family: "Cookie", cursive;
+  font-size: 41px;
+  color: var(--deep-blue);
+  position: absolute;
+  left: 100px;
+  bottom: 30px;
```
Comment (soerendolling) [inline client/src/components/friends-page/FriendsContent.css:72]: For example here its from inside to outside, which i think is fine. I would just stick with one sorting through the rulesets

**Review by soerendolling — APPROVED** [review]

File: client/src/components/friends-page/FriendsContent.css (hunk 1 above)
Comment (hugh-burgess) [reply client/src/components/friends-page/FriendsContent.css]: i like this strategy! will consider...

**Pull request merged**